install.sh

```diff
@@ -755,8 +755,7 @@ if [ $targetSystem = raspi ]
 then
     echo "Preparing USB automount"
     # systemd-udevd creates its own filesystem namespace, so mount is done, but it is not visible in the principal namespace.
-    sudo mkdir /etc/systemd/system/systemd-udevd.service.d/
-    echo -e "[Service]\nMountFlags=shared" | sudo tee /etc/systemd/system/systemd-udevd.service.d/mountFlagOverride.conf
+    sudo sed -i -e '/^PrivateMounts/ s/yes/no/' /lib/systemd/system/systemd-udevd.service
```

Comment:
Does the override mechanism not work anymore? Wouldn't
```
echo -e "[Service]\nPrivateMounts=no" | sudo tee /etc/systemd/system/systemd-udevd.service.d/udev-service-override.conf
```
work, too? It would allow us to NOT change system files that will get updated with the next systemd package update.

WDYT?

Reply:
ah, My bad
I was so indulged in finding why it was not working, I totally forgot about the override method.
updated the PR now.
Thank You @hongquan and @norbusan

Reply:
Thanks for the update